vcl_templates/www.vcl.erb

```diff
@@ -296,6 +296,9 @@ sub vcl_deliver {
     add resp.http.Set-Cookie = "ABTest-Example=" req.http.GOVUK-ABTest-Example "; expires=" now + 1d;
   }
 
+  # Set the tls version cookie with the raw protocol version from fastly
+  add resp.http.Set-Cookie = "TLSversion=" tls.client.protocol "; max-age=0; secure";
```

Comment:
Do we need an explicit expiry, or can we just use a session cookie (by excluding both `max-age` and `expires`)?  We can also check for a pre-existing cookie before setting a new one:

```js
if (req.http.Cookie !~ "TLSversion") {
  add resp.http.Set-Cookie = "TLSversion=" tls.client.protocol "; secure";
}
```